jetcd-core/src/main/java/com/coreos/jetcd/data/ByteSequence.java

```diff
@@ -67,7 +67,7 @@ public boolean equals(Object obj) {
     }
   }
 
-  protected ByteString getByteString() {
+  public ByteString getByteString() {
```

Comment:
This method should not be made public as you can then change the content of the `byteString` which invalidates the hash code calculation done in the constructor.

Reply:
According to the [Javadoc](https://developers.google.com/protocol-buffers/docs/reference/java/com/google/protobuf/ByteString) ByteString is immutable like the String class. So a change should not be possible.

Reply:
Ah correct, I got confused by some methods like concat